### PR TITLE
dev-scripts: add serial tests

### PR DESCRIPTION
**Pull request by andfasano**

### CODE REVIEW

File: ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml [hunk 1]
```diff
@@ -50,11 +50,23 @@ tests:
     workflow: baremetalds-e2e-ovn-ipv6
 - as: e2e-metal-ipi-ovn-dualstack
   optional: true
+  skip_if_only_changed: (^(docs|.github)/)|(\.md$)|((^|/)OWNERS$)
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e-ovn-dualstack
+- as: e2e-metal-ipi-serial-ipv4
+  skip_if_only_changed: (^(docs|.github)/)|(\.md$)|((^|/)OWNERS$)
```
Comment (elfosardo) [inline ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml:58]: let's also add DCO and LICENSE here and in all the other skip_if_only_changed entries

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml (hunk 1 above)
Comment (andfasano) [reply ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml:58]: Good idea, thanks

**Comment by andfasano** [comment]
Requires https://github.com/openshift/release/pull/24208

**Review by elfosardo — APPROVED** [review]

**Pull request merged**